src/utils/components/CPUMemoryModal/components/CPUInput/CPUMemoryModalHeader.tsx

```diff
@@ -0,0 +1,32 @@
+import React, { FC } from 'react';
+
+import { useKubevirtTranslation } from '@kubevirt-utils/hooks/useKubevirtTranslation';
+import { Button, ButtonVariant, Popover, Title, TitleSizes } from '@patternfly/react-core';
+import { HelpIcon } from '@patternfly/react-icons';
+
+import './CPUMemoryModalHeader.scss';
+
+const CPUMemoryModalHeader: FC = ({}) => {
+  const { t } = useKubevirtTranslation();
+
+  return (
+    <>
+      <Title
+        className="cpu-memory-modal-header"
+        headingLevel="h4"
+        id="cpu-memory-modal-title"
+        size={TitleSizes.md}
```

Comment:
whats the difference between size and headinglevel?

Reply:
I just looked. They map to each other. h4 maps to md, so I'll remove the size.